px-map-behavior-root.es6.js

```diff
@@ -400,6 +400,17 @@
       },
 
       /**
+       * Set to disable zooming on double-click.
```

Comment:
Did we confirm if this works on only desktop, or if its desktop and mobile? If its only desktop, can you add that it "only works on desktop"?

Reply:
works on mobile 👍